content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -109,6 +109,10 @@ to manage the lifecycle of Istio.
     $ helm install install/kubernetes/helm/istio --name istio --namespace istio-system --set global.mtls.enabled=true
     {{< /text >}}
 
+## Installation Options
```

Comment:
For headers, only capitalize the first word. So "Installation options"

Reply:
I removed this section and added it to the previous section as suggested by Lin